src/utils/makeReactNativeConfig.js

```diff
@@ -31,6 +31,7 @@ type ConfigOptions = {|
   providesModuleNodeModules?: (string | { name: string, directory: string })[],
   hasteOptions?: *,
   initializeCoreLocation?: string,
+  includedNodeModules?: Array<string>,
```

Comment:
Shouldn't this be `excludedNodeModules`?

Reply:
The way things have been implemented before this PR is not the typical "include all node modules except the ones listed for exclusion", but rather a "include none of the node modules except the ones listed for exclusion."

I haven't changed the paradigm, I've just exposed these as a configurable option.